php/php.editor/src/org/netbeans/modules/php/editor/model/impl/ClassScopeImpl.java

```diff
@@ -335,6 +336,13 @@ public Collection<? extends FieldElement> getInheritedFields() {
                 allFields.add(new FieldElementImpl(traitScope, field));
             }
         }
+        // GH-4683 get fields of mixin
+        Set<TypeMemberElement> mixinTypeMembers = index.getAccessibleMixinTypeMembers(this, this);
+        for (TypeMemberElement mixinTypeMember : mixinTypeMembers) {
+            if (mixinTypeMember instanceof org.netbeans.modules.php.editor.api.elements.FieldElement) {
+                allFields.add((new FieldElementImpl(this, (org.netbeans.modules.php.editor.api.elements.FieldElement) mixinTypeMember)));
+            }
```

Comment:
how about ?
index.getAccessibleMixinTypeMembers(this, this).stream().
     filter(m -> m instanceof FieldElement).
     map(m -> new FieldElementImpl(this, (FieldElement) m).
     forEach(allFieds::add).

Reply:
I don't want to do it. Maybe, hard to debug.

Reply:
I tend to agree - streams are nice and useful for simple things. But quite difficult to debug in case of problems 😅

Reply:
just a loose suggestions, feel free to ignore :)

Reply:
Thanks. Nothing personal, of course ☺️

Reply:
off-topic: any ideas to improve debugging of lambda expressions?

Reply:
I am not sure but I think that IntelliJ IDEA has some special "view" fir it when debugging. And also, a hint to convert streams into loops (and vice versa).